stratum/hal/lib/tdi/dpdk/dpdk_port_manager.cc

```diff
@@ -86,19 +80,6 @@ ::util::Status DpdkPortManager::GetPortCounters(int device, int port,
   return ::util::OkStatus();
 }
 
-::util::Status DpdkPortManager::RegisterPortStatusEventWriter(
```

Comment:
Do the relevent MOCK routines need to be cleaned up in the dpdk_port_manager_mock.h?

Reply:
Same comment for es2k and tofino targets

Reply:
I don't think so. `Es2kPortManagerMock` is based on `Es2kPortManager`. It mocks all the methods the class supports, including the ones it inherits from `TdiPortManager`. The same is true of the other PortManagerMocks. I believe this is correct.